graspologic/cluster/autogmm.py

```diff
@@ -515,11 +530,27 @@ def fit(self, X, y=None):
             linkage=self.linkage,
             covariance_type=self.covariance_type,
             n_components=range(lower_ncomponents, upper_ncomponents + 1),
-            random_state=[self.random_state],
         )
         param_grid = list(ParameterGrid(param_grid))
+        # Append multiple_init rows if desired
+        if self.n_init is not None and self.label_init is None:
+            multiple_init = [
+                params.copy()
+                for params in param_grid
```

Comment:
I am not sure why the part of adding more inits is moved here. It seems to me that by doing this, the `param_grid` will be looped through once more here (it also gets looped through in `_process_paramgrid`)

Reply:
Hey @tliu68, sorry for the late response! I did move it here, because I remember you had requested to remove the creation of gm_params_processed inside process_paramgrid. I can switch back to the previous implementation if needed

Reply:
Well, I don't see a reason for it to be moved here. Maybe I didn't make it clear.